src/licenses/LicenseCategory.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TURTLEBROWSER_LICENSECATEGORY_H
+#define TURTLEBROWSER_LICENSECATEGORY_H
+
+#include <QObject>
+
+namespace licenses {
+
+  enum class LicenseCategory {
```

Comment:
you might write:

`enum class LicenseCategory: uint8_t`

in case you need to cast it to integer, in case you need it

Reply:
Hm, I assumed that the default underlying type was int, but I can't seem to see that that is defined

Reply:
Since I couldn't immediately find that the default underlying type was int (I don't know where I got that impression) I changed it like you said.